chrome/content/zotero/integration/citationDialog/keyboardHandler.mjs

```diff
@@ -86,12 +86,17 @@ export class CitationDialogKeyboardHandler {
 	_handleTopLevelKeydown(event) {
 		let handled = false;
 		let tgt = event.target;
-		let isKeyboardClickable = tgt.classList.contains("keyboard-clickable") || tgt.tagName.includes("button");
-		// Space/Enter will click on a button or keyboard-clickable components
+		let isKeyboardClickable = tgt.classList.contains("keyboard-clickable") || (Zotero.isWin && tgt.tagName.includes("button"));
+		// Space/Enter will click on keyboard-clickable components
```

Comment:
Move back above (and probably explain the Windows vs. Mac thing)

Reply:
It took me a second to phrase it - how about this now:
```
  // Space/Enter will click on keyboard-clickable components.
  // On macOS, focused buttons are only clickable with Space (not Enter),
  // and on Windows they are clickable with both.
  // This distinction determines if Enter on a button in a panel will perform a click or close it.
```